### ISSUE TITLE
A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_ScreenReader: Status message not announced by screen reader for the content that is updated on the screen.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9471]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yWCAG2.1;#A11yTCS;#SQL Azure Data Studio;#DesktopApp;#MacOs;#Benchmark;#A11y_SQLAzureDataStudioBenchmark;#A11ySev1;#MAS4.1.3;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Screen reader: Voice Over
MAS reference: MAS4.1.3
Mac OS:10.15.3

**Repro Steps:**

Launch Azure Data Studio.
Select the Connections icon (if not already selected) from left bar.
Hover over the 'Servers' header and select the 'New Connection' icon.
Navigate to connection details.
Launch NVDA and navigate till connect button and press enter.
Check if proper status message is announced by screen reader for all the data that is updated on the screen.

**Actual:**
Status message about data that is updated on the screen is not announced by the screen reader.

**Expected:**
Status message should be announced for all the content that is displayed after navigating to any of the controls present on the screen.

**User Impact:**
Screen reader users might not come to know what is the data that is being updated on the screen.

**Attachment link reference:**
[9471_A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_ScreenReader.zip](https://github.com/microsoft/azuredatastudio/files/4296600/9471_A11y_SQLAzureDataStudio_ConnectToASQLServer_AdvancedProperties_ScreenReader.zip)

**Comment by alanrenmsft** [comment]
set aria-description to the input elements on advanced properties page.

**Comment by Charles-Gagnon** [comment]
This has been fixed

**Comment by fsteffi** [comment]
GitHubTags:#A11y_SQlADS_WCAG2.1;

**Comment by aniketr99** [comment]
Verified the bug in below environment:
Version: 1.41.1 (Universal)
Commit: https://github.com/microsoft/azuredatastudio/commit/33dce95b152227c4f10f6251ab30690fe9895c0c
Date: 2023-01-28T00:49:19.454Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Darwin arm64 22.3.0
issue is fixed hence adding #Closed; tag.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-SQLADS-Win32-Feb2020;
